src/ringct/multiexp.cc

```diff
@@ -450,11 +449,9 @@ rct::key straus(const std::vector<MultiexpData> &data, const std::shared_ptr<str
   STEP = STEP ? STEP : 192;
 
   MULTIEXP_PERF(PERF_TIMER_START_UNIT(setup, 1000000));
-  static constexpr unsigned int mask = (1<<STRAUS_C)-1;
```

Comment:
It is actually used inside `#elif 1` condition below.

Reply:
Good catch, overlooked that one.